src/Batch/Client/changelog.md

```diff
@@ -1,16 +1,23 @@
 ## Azure.Batch release notes
 
+### Changes in 6.1.0
+#### REST API version
+This version of the Batch .NET client library targets version 2017-01-01.4.0 of the Azure Batch REST API.
+
+#### Packaging
+- The client library is now supported on .NET Core. In addition to a `net45` assembly, the package now includes a `netstandard1.5` assembly. The `net45` assembly is identical to the one from `6.0.0`.
+
```

Comment:
@tamram Updated to use your wording. We are on 6.0.0 now and we are releasing 6.1.0 (no changes except enabling netcore), so I am just telling people that if they are using net45, 6.0.0 == 6.1.0. The only change is the new netcore stuff.